crates/rnote-ui/src/appwindow/mod.rs

```diff
@@ -104,6 +104,30 @@ impl RnAppWindow {
         self.imp().overlays.get()
     }
 
+    pub(crate) fn get_number_of_tabs(&self) -> u32 {
+        self.imp().overlays.tabview().pages().n_items()
+    }
+
+    pub(crate) fn get_tab_i(&self, i: u32) -> Result<RnCanvas, anyhow::Error> {
```

Comment:
Instead of doing it this way why not write a function that returns all tabs in order as a vec?

Reply:
Ok, I switched the function.